rust/src/applayertemplate/template.rs

```diff
@@ -200,6 +200,7 @@ impl TemplateState {
                     start = rem;
 
                     if let Some(tx) =  self.find_request() {
+                        tx.tx_data.updated[1] = true;
```

Comment:
I don't really like this indexing with just a number to indicate a direction. Can we either index by some name or use something like `updated_ts` / `updates_tc`